packages/web-components/fast-components/src/design-system-provider/README.md

```diff
@@ -0,0 +1,260 @@
+# Design System Provider
+- [What is a Design System](#what-is-a-design-system)
```

Comment:
I'm not sure that we should put an inline ToC as that will make the final Docusaurus generated docs a bit odd. Docusaurus will generate the in-page ToC as a right sidebar.

Reply:
That's true, I forgot about docusaurus when I made the comment requesting it. Should we come to a consensus about how to deal with this? As of right now I have TOC on `fast-tooling` and `fast-tooling-react` packages because looking at the README by itself (on npm or in github) without it is cumbersome.

Reply:
okay - I'll remove.